src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -120,34 +120,39 @@ function ReportActionItemFragment(props) {
                 const editedTag = props.fragment.isEdited ? `<edited ${isPendingDelete ? 'deleted' : ''}></edited>` : '';
-                const htmlContent = applyStrikethrough(html + editedTag, isPendingDelete);
+                const htmlContent = isPendingDelete ? `<del>${html}</del>` : html;
 
-                return <RenderHTML html={props.source === 'email' ? `<email-comment>${htmlContent}</email-comment>` : `<comment>${htmlContent}</comment>`} />;
+                const htmlWithTag = editedTag ? `${htmlContent}${editedTag}` : htmlContent;
+
+                return <RenderHTML html={props.source === 'email' ? `<email-comment>${htmlWithTag}</email-comment>` : `<comment>${htmlWithTag}</comment>`} />;
             }
             const containsOnlyEmojis = EmojiUtils.containsOnlyEmojis(text);
 
             return (
-                <Text
-                    selectable={!DeviceCapabilities.canUseTouchScreen() || !props.isSmallScreenWidth}
-                    style={[containsOnlyEmojis ? styles.onlyEmojisText : undefined, styles.ltr, ...props.style]}
-                >
-                    {convertToLTR(text)}
+                <Text style={[containsOnlyEmojis ? styles.onlyEmojisText : undefined, styles.ltr, ...props.style]}>
+                    <Text
+                        selectable={!DeviceCapabilities.canUseTouchScreen() || !props.isSmallScreenWidth}
+                        style={[containsOnlyEmojis ? styles.onlyEmojisText : undefined, styles.ltr, ...props.style, isPendingDelete && styles.offlineFeedback.deleted]}
+                    >
+                        {convertToLTR(text)}
+                    </Text>
                     {Boolean(props.fragment.isEdited) && (
-                        <Text
-                            fontSize={variables.fontSizeSmall}
-                            color={themeColors.textSupporting}
-                            style={[editedLabelStyles, ...props.style]}
-                        >
+                        <>
                             <Text
                                 selectable={false}
-                                style={[containsOnlyEmojis ? styles.onlyEmojisTextLineHeight : undefined, styles.w1, styles.userSelectNone]}
+                                style={[containsOnlyEmojis ? styles.onlyEmojisTextLineHeight : undefined, styles.userSelectNone]}
                             >
                                 {' '}
                             </Text>
-                            {props.translate('reportActionCompose.edited')}
-                        </Text>
+                            <Text
+                                fontSize={variables.fontSizeSmall}
+                                color={themeColors.textSupporting}
+                                style={[editedLabelStyles, isPendingDelete && styles.offlineFeedback.deleted, ...props.style]}
```

Comment:
I see 2 approaches here for conditional styles. Let's keep consistence. 
```suggestion
                                style={[editedLabelStyles, isPendingDelete ? styles.offlineFeedback.deleted : undefined, ...props.style]}
```